cnd/src/swap_protocols/hbit/events.rs

```diff
@@ -0,0 +1,77 @@
+use crate::{
+    asset, htlc_location,
+    swap_protocols::{hbit::HtlcParams, Secret},
+    transaction,
+};
+use chrono::NaiveDateTime;
+use serde::{Deserialize, Serialize};
+
+// TODO: Remove the Herc20 stuff i.e., no separate Deployed/Funded states.
```

Comment:
This TODO can go away if you resolve my comment in `cnd/src/swap_protocols/hbit.rs`

Reply:
Removed, thanks.